CreeDictionary/CreeDictionary/views.py

```diff
@@ -86,18 +103,54 @@ def search_results(request, query_string: str):  # pragma: no cover
     )
 
 
-def lemma_details_internal(request, lemma_id: int):  # pragma: no cover
+@require_GET
+def lemma_details_internal(request):
     """
-    render paradigm table for a lemma
+    Render paradigm.html for a lemma. `index` view function renders a whole page that contains paradigm.html too.
+    This function, however, is used by javascript to dynamically replace the paradigm with the ones of different sizes.
+
+    `lemma-id` and `paradigm-size` are the expected query params in the request
+
+    4xx errors will have a single string as error message
+
+    :raise 400 Bad Request: when the query params are not as expected or inappropriate
```

Comment:
I hesitated between 400 and 422 Unprocessable Entity. Decided to use 400 cuz Django doesn't have a shortcut for 422

Reply:
Haha, fair enough!